packages/web-components/src/stories/tooltip.stories.mdx

```diff
@@ -0,0 +1,242 @@
+import { extractArgTypes } from '@astrouxds/storybook-addon-docs-stencil'
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+import { html, render } from 'lit-html'
+
+<Meta
+    title="Components/Tooltip"
+    component="rux-tooltip"
+    argTypes={extractArgTypes('rux-tooltip')}
+    parameters={{
+        actions: {
+            handles: [
+                'ruxtooltipopened rux-tooltip',
+                'ruxtooltipclosed rux-tooltip'
+            ],
+        },
+    }}
+/>
+
+# Tooltip
+
+A Tooltip is a message box that provides a concise amount of relevant contextual information about an associated UI element. Tooltips are activated by hovering over, or focusing on an element.
+
+export const Default = (args) => {
+    return html`
+<div style="padding: 5%; display: flex; flex-direction: column; justify-content: center; align-items: center;">
+    <rux-tooltip
+        message="${args.message}"
+        placement="${args.placement}"
+        delay="${args.delay}"
+        open="${args.open}"
```

Comment:
define strategy: absolute here and wire up to tooltip. same for offset and disable-auto-update so the controls work

Reply:
All set!